lib/backup/notifier/mail.rb

```diff
@@ -230,17 +230,3 @@ def new_email
     end
   end
 end
-
-# Patch mail v2.5.4 Exim delivery method
-# https://github.com/backup/backup/issues/446
-# https://github.com/mikel/mail/pull/546
-module Mail
-  class Exim
-    def self.call(path, arguments, _destinations, encoded_message)
-      popen "#{path} #{arguments}" do |io|
-        io.puts encoded_message.to_lf
-        io.flush
-      end
-    end
-  end
-end
```

Comment:
Ugh, my bad. I somehow had it in my head that it had been merged and wasn't needed. My apologies! I've reverted that change. This should be good to go now.